x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/crud/read_rules.ts

```diff
@@ -15,6 +15,7 @@ import type {
 import { withSecuritySpan } from '../../../../../utils/with_security_span';
 import type { RuleParams } from '../../../rule_schema';
 import { isAlertType } from '../../../rule_schema';
```

Comment:
have you had a chance to test bulk edit action? Does it migrate investigation filed for free? Or some work is needed there?

Reply:
I can see test for bulk_update, which is deprecated in favour of bulk_actions(bulk_edit - add/removed tags/actions,schedule, etc)

Tests for that one can be found in `x-pack/test/detection_engine_api_integration/security_and_spaces/group10/perform_bulk_action.ts`
I am interested in that one, because it fetches rules within alert framework for editing, so rules are not read through find API, which is covered.

Reply:
Thanks for pointing this out. I'll add tests there now.